apps/labs/posts/alt-text-scipy-2022.md

```diff
@@ -0,0 +1,72 @@
+
+## The scavenger hunt
+
+First, we need to acknowledge the four projects participating in this pilot:
+
+1. Project Jupyter[^1]
+2. SciPy
+3. Dask
+4. Bokeh
+
+Thank y'all for taking the time to prepare for the scavenger hunt, and wanting to improve your documentation.
+
+## How to participate.
+
+- Follow each link/code to a pull request.
+- Follow the pull request instructions to modify the respective alt text in markdown and rst formats.
+
+| JupyterLab [^1]     | SciPy                 | Dask                | Bokeh                 |
+| ------------------- | --------------------- | ------------------- | --------------------- |
+| [![][jlabqr]][jlab] | [![][scipyqr]][scipy] | [![][daskqr]][dask] | [![][bokehqr]][bokeh] |
+
+## Why participate?
+
+- _low barrier to entry_ to participate in open source projects
+- _improve the accessibility of open source documentation_
+- _learn about more alt text_
+
+## What next?
+
+- Start submitting and improving your alt text!
+  - [Working Jupyter alt text guide](https://github.com/Quansight-Labs/jupyter-accessibility-workshops/blob/main/docs/alt-text-guide.md#diagrams-charts-plots)
+  - [Chartability heuristics for accessible visualizations](https://chartability.fizz.studio/)
+- Stay tuned later this week to learn how you fight inaccessibility with our zine release.
+- Use more alt text in your social media and documentation.
+- Give the project maintainers time to merge your submissions.
+- Enjoy [__How to Write Alt Text for Scientific Diagrams and Visualization__][zine]
+
+[jlab]: https://github.com/isabela-pf/jupyterlab/pull/1
+[jlabqr]: https://user-images.githubusercontent.com/4236275/176369870-6fd6501f-288d-48c5-8cc3-4eaad0a35ee1.png
+[scipy]: https://github.com/scipy/scipy/pull/16546
+[scipyqr]: https://user-images.githubusercontent.com/4236275/177211155-c9ef9a49-962d-429e-83ff-5ef1e9f96ca3.png
+[dask]: https://github.com/pavithraes/dask/pull/1
+[daskqr]: https://user-images.githubusercontent.com/4236275/177203555-1e0ef684-8113-4150-8909-2996cfbe060e.png
+[bokeh]: https://github.com/bokeh/bokeh/pull/12211
+[bokehqr]: https://user-images.githubusercontent.com/4236275/177208706-304e0176-969e-48a0-becd-730a8c92eb01.png
+[placeholder]: #
+[placeholderqr]: https://chart.googleapis.com/chart?cht=qr&chl=https%3A%2F%2Fgithub.com%2Falt-text-task-force&chs=300x300&choe=UTF-8&chld=L|2
+[zine]: https://heyzine.com/flip-book/f47006ad71.html
+
+[^1]: These alt text events, that started in Project Jupyter, have been a community mainstay for over a year. In these [low-code workshops](https://labs.quansight.org/blog/2021/09/low-code-contributions-through-github) we have been able to remove the complexities of git in contributing to open source. These events have been run [in the past with several communities](https://github.com/isabela-pf/a11y-events/tree/main/workshop-resources/alt-text#past-events).
```

Comment:
The "and thus expand participation" is optional -- just think the thought is ~incomplete without it.

```suggestion
[^1]: These alt text events, which started in Project Jupyter, have been a community mainstay for over a year. In these [low-code workshops](https://labs.quansight.org/blog/2021/09/low-code-contributions-through-github) we have been able to remove the complexities of git in contributing to open source and thus expand participation. These events have been run [in the past with several communities](https://github.com/isabela-pf/a11y-events/tree/main/workshop-resources/alt-text#past-events).
```